recipes/econ-ark/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "econ-ark" %}
+{% set version = "0.10.0.dev2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d14170e9a70d95c9380bc1bc16b73876d88a0c86d37e293ae84c6d052148d8f2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

The package doesn't involve compiled binaries, so this can be `noarch: python`
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
  noarch: python
```